src/jsutils/didYouMean.ts

```diff
@@ -12,6 +12,10 @@ export function didYouMean(
   firstArg: string | ReadonlyArray<string>,
   secondArg?: ReadonlyArray<string>,
 ) {
+  if (process.env.NODE_ENV === 'production') {
```

Comment:
@saihaj Thanks for the review, I wonder if by default the library should enforce good practices so that we do not leave vulnerable code in production systems?

Reply:
Yes but this isn’t a platform agnostic fix. This library can be used in non-node environments. So it is something in the environment you are implementing that this can be handled.

Reply:
Understood. Keen to see how this progress and I could help with the fix: https://github.com/graphql/graphql-js/issues/2247#issuecomment-962397993